.github/workflows/pr-validate-label-teamlabel.yml

```diff
@@ -1,12 +1,9 @@
-name: "Validate PRs"
-
+name: "Validate PRs & Assign PR team labels & Pull Request Labeler"
 on:
+  pull_request:
+    branches:
+      - main
   pull_request_target:
-    types:
```

Comment:
As per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target we need to have these types otherwise, it may not trigger for `edited` action.
![image](https://github.com/calcom/cal.com/assets/1780212/d228759a-f7c1-43c9-94d4-18487f4450cc)